content/standards/index.html.md.erb

```diff
@@ -21,7 +21,7 @@ The Data Standards Catalogue currently contains the following Standards:
     </thead>
     <tbody class="govuk-table__body">
       <% current_resource.children.sort_by{ |page| get_link_text(page).downcase }.each do |f| %>
-        <% if f.content_type.include? "html" %>
+        <% if f.path.include? "index.html" %>
```

Comment:
Alternatively, we could add something to the front matter to say `exclude_from_list`?

Reply:
good point - have tried your suggestion, is this what you meant? https://github.com/alphagov/data-standards-authority/pull/100/commits/128e3a5927a90720bed76290b684cbf626027024